go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/basemachina/bridge
 
-go 1.21.9
+go 1.22.4
```

Comment:
For use Routing Enhancements that added in Go 1.22
https://go.dev/blog/routing-enhancements